apps/client/src/module/auth/auth-form-container.tsx

```diff
@@ -6,12 +6,12 @@ type Props = {
 export function AuthFormContainer(props: Props) {
   return (
     <form
-      className="dark:bg-primary-900 bg-primary-50 flex w-[520px] max-w-[520px] flex-col gap-5 px-[60px] py-[50px] transition-colors duration-300"
+      className="dark:bg-primary-900 bg-primary-50 flex w-full max-w-[520px] flex-col gap-5 px-[60px] py-[50px] transition-colors duration-300 max-sm:px-[30px] max-sm:py-[35px] lg:my-5 lg:mt-10 lg:max-w-[1000px] lg:px-[80px] lg:py-[70px]"
```

Comment:
hmmmm, I don't think we need `lg:my-5 lg:mt-10 lg:max-w-[1000px] lg:px-[80px] lg:py-[70px]`

just let whatever setting we have at md-breakpoint does the job is fine.

Reply:
yeah sorry.... wasn't expecting this whole ultrawide thingy would take this long. 

the idea is to have the ui maintain it's md-breakpoint look when the screen width gets bigger, there shouldn't be any specific styling tied to lg-breakpoint. because 
 A. your ui can scale on its own
 B. there's no design for larger viewport, hence md-styles should be kept the same